lib/api/controllers/authController.js

```diff
@@ -7,26 +7,46 @@ module.exports = function AuthController (kuzzle) {
 
   this.passport = new PassportWrapper(kuzzle);
 
-  this.login = function (requestObject) {
+  this.logout = function (requestObject, context) {
+    var
+      deferred = q.defer(),
+      response;
+
+    kuzzle.pluginsManager.trigger('auth:logout', context)
+      .then(() => {
+        return kuzzle.repositories.token.expire(context.token);
+      })
+      .then(() => {
+        response = new ResponseObject(requestObject);
+        deferred.resolve(response);
+      })
+      .catch(function (err) {
+        deferred.reject(err);
+      });
+
+    return deferred.promise;
+  };
+
+  this.login = function (requestObject, context) {
```

Comment:
Same remarks than for the `logout` method:

- (nitpicking) Using a deferred promise is not necessary
- In case of rejection, must return a `ResponseObject`